wom/src/test/scala/wdl4s/wdl/wom/WdlSubworkflowWomSpec.scala

```diff
@@ -0,0 +1,67 @@
+package wdl4s.wdl.wom
+
+import cats.data.Validated.{Invalid, Valid}
+import org.scalatest.{FlatSpec, Matchers}
+import wdl4s.wdl.{ImportResolver, WdlNamespace, WdlNamespaceWithWorkflow}
+import wdl4s.wom.graph.Graph
+
+class WdlSubworkflowWomSpec extends FlatSpec with Matchers {
+
+  behavior of "WdlNamespaces with subworkflows"
+
+  it should "support WDL to WOM conversion of subworkflows in scatters" in {
+    val outerWdl =
+      """import "import_me.wdl" as import_me
+        |
+        |workflow outer {
+        |  Array[Int] xs
+        |  scatter (x in xs) {
+        |    call import_me.inner as inner { input: i = x }
+        |  }
+        |  output {
+        |    Array[String] outs = inner.out
+        |  }
+        |}""".stripMargin
+
+    val innerWdl =
+      """task foo {
+        |  Int i
+        |  command {
+        |    echo ${i}
+        |  }
+        |  output {
+        |    String out = read_string(stdout())
+        |  }
+        |}
+        |
+        |workflow inner {
+        |  Int i
+        |  call foo { input: i = i }
+        |  output {
+        |    String out = foo.out
+        |  }
+        |}
+      """.stripMargin
+
+
+    def innerResolver: ImportResolver = _ => innerWdl
+
+    val namespace = WdlNamespace.loadUsingSource(
+      workflowSource = outerWdl,
+      resource = None,
+      importResolver = Some(Seq(innerResolver))).get.asInstanceOf[WdlNamespaceWithWorkflow]
+    import lenthall.validation.ErrorOr.ShortCircuitingFlatMap
+
+    val outerWorkflowGraph = namespace.womExecutable.flatMap(_.graph)
+
+    outerWorkflowGraph match {
+      case Valid(g) => validateGraph(g)
+      case Invalid(errors) => fail(s"Unable to build wom version of workflow with subworkflow from WDL: ${errors.toList.mkString("\n", "\n", "\n")}")
+    }
+
+    def validateGraph(workflowGraph: Graph): Unit = {
+      ()
+    }
```

Comment:
ToL: What is this method providing?

Reply:
Warm fuzzy feelings without actually validating anything....?

Reply:
I'm going to add a couple more tests here, hold on...